### DISCUSSION TITLE
v1.2.0 to add support to WiFi, WiFiNINA, WiFi101, ESP WiFi, ESP8266-AT WiFi, etc.

### DISCUSSION THREAD

**Discussion by khoih-prog** [khoih-prog/MDNS_Generic#7] [Announcements]
### Releases v1.2.0

 1. Add support to RP2040-based boards, such as **Nano_RP2040_Connect, RASPBERRY_PI_PICO, ADAFRUIT_FEATHER_RP2040 and GENERIC_RP2040**, using [**Arduino-mbed RP2040** core](https://github.com/arduino/ArduinoCore-mbed) using WiFi / [WiFiNINA_Generic Library](https://github.com/khoih-prog/WiFiNINA_Generic)
 2. Add support to RP2040-based boards, such as **RASPBERRY_PI_PICO, ADAFRUIT_FEATHER_RP2040 and GENERIC_RP2040**, using [**Earle Philhower's arduino-pico** core](https://github.com/earlephilhower/arduino-pico) using WiFi / [WiFiNINA_Generic Library](https://github.com/khoih-prog/WiFiNINA_Generic)
 3. Add support to ESP32 and ESP8266 boards using native ESP WiFi
 4. Add support to Generic boards, such as **MKR WiFi1010**, using [WiFi101 Library](https://github.com/arduino-libraries/WiFi101)
 5. Add WiFi-related examples. 
 6. Modify hostname from easily-duplicated `arduino` to board-related hostname to fix issue caused by duplicated hostname, such as [mDNS stops working after two minutes #5](https://github.com/khoih-prog/MDNS_Generic/issues/5)
 7. Verify issue [**mDNS stops working after two minutes** #5](https://github.com/khoih-prog/MDNS_Generic/issues/5) is fixed, using Nano_RP2040_Connect, WiFiNINA FW 1.4.8 and [WiFiNINA_Generic Library](https://github.com/khoih-prog/WiFiNINA_Generic)
 8. Change License from `MIT` to [GPLv3](https://github.com/khoih-prog/MDNS_Generic/blob/master/LICENSE)

<hr /><em>This discussion was created from the release <a href='https://github.com/khoih-prog/MDNS_Generic/releases/tag/v1.2.0'>v1.2.0 to add support to WiFi, WiFiNINA, WiFi101, ESP WiFi, ESP8266-AT WiFi, etc.</a>.</em>

**Comment by stefat** [comment]
Great ! I’ll try immediately. Thanks.
